src/kubelet/metric/metric.go

```diff
@@ -330,6 +330,16 @@ func FromRawGroupsEntityTypeGenerator(groupLabel string, rawEntityID string, gro
 	}
 }
 
+func FromLabelGetNamespace() definition.NamespaceGetterFunc {
+	return func(metrics definition.RawMetrics) string {
+		if ns, ok := metrics["namespace"].(string); ok {
+			return ns
+		}
+
+		return ""
+	}
+}
```

Comment:
The same that the previous comment in `prometheus.definition.go` about the namespace getter function definition.

Reply:
Done in last commit